testing/trino-server-dev/etc/config.properties

```diff
@@ -14,12 +14,10 @@ http-server.http.port=8080
 
 discovery.uri=http://localhost:8080
 
-exchange.http-client.max-connections=1000
 exchange.http-client.max-connections-per-server=1000
 exchange.http-client.connect-timeout=1m
 exchange.http-client.idle-timeout=1m
 
-scheduler.http-client.max-connections=1000
```

Comment:
related?

Reply:
nope, just removing defunct properties to make dev server config work again